src/test/java/com/google/sps/UserTest.java

```diff
@@ -0,0 +1,89 @@
+package com.google.sps;
+
+import com.google.sps.user.User;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public final class UserTest {
+    private static final String ID = "userid";
+    private static final String EMAIL = "user@gmail.com";
+    private static final String NAME = "username";
+    private static final String SELF_INTRODUCTION = "I am the user";
+    private static final String IMG_URL = "/img.com";
+
+    @Test
+    public void getId() {
+        User newUser = new User.Builder(ID, EMAIL).build();
+        Assert.assertEquals(newUser.getId(), ID);
+    }
+
+    @Test
+    public void getEmail() {
+        User newUser = new User.Builder(ID, EMAIL).build();
+        Assert.assertEquals(newUser.getEmail(), EMAIL);
+    }
+
+    @Test
+    public void getName() {
+        // User with no preset name.
+        User newUser = new User.Builder(ID, EMAIL).build();
+        Assert.assertEquals(newUser.getName(), null);
```

Comment:
Maybe for each of these test cases, we can add different test methods so if an error occurred, we can easily find the error occurred.

I use this convention for naming the test method (given by Chris):
`<methodUnderTest>_<condition>_<expectedOutcome>`

Reply:
Applied naming convention.